images/logging-eventrouter.yml

```
@@ -1,15 +1,24 @@
+content:
```

Since this is a golang build we do need the following:
```suggestion
container_yaml:
  go:                          
    modules:
    - module: github.com/openshift/eventrouter
content:
```